src/pyosmeta/parse_issues.py

```diff
@@ -572,9 +579,6 @@ def get_issue_meta(
         """
         issue_meta = {}
         for item in body_data[0:end_range]:
-            # Clean date accepted element
```

Comment:
this was just a thing i added in a really random place. it worked but it is so hard to find it if you're working on the code.